integration-tests/test/native-eth-ovm-calls.spec.ts

```diff
@@ -38,7 +38,7 @@ describe('Native ETH value integration tests', () => {
     }
 
     const value = ethers.utils.parseEther('0.01')
-    await fundUser(env.watcher, env.l1Bridge, value, wallet.address)
+    await fundUser(env.messenger, env.l1Bridge, value, wallet.address)
```

Comment:
Note that `fundUser` has already been patched to use the SDK in a recent commit, you may need to make a second PR or rebase.